src/classes/frSetupController.cls

```diff
@@ -90,7 +178,7 @@ public with sharing class frSetupController {
 			}
 		}
 		donationSFOptions.sort();
-		for(frField__mdt field : [SELECT DeveloperName, MasterLabel FROM frField__mdt WHERE Type__c = :DONATION_TYPE ORDER BY MasterLabel]) {
+		for(frField__mdt field : [SELECT DeveloperName, MasterLabel FROM frField__mdt WHERE Type__c = :DONATION_TYPE OR Type__c = 'All' ORDER BY MasterLabel]) {
```

Comment:
How does a field of Type ALL work here?  Generally, the field definitions are managed per the type of entities that sync and are mappable from Funraise, is there a type of field coming from Funraise that would exist on all mappable entities?

Reply:
I wanted to follow up on this one as I'm not following the reasoning for a field having a type__c of ALL

Reply:
The type ALL is just used to get the funraise id to populate as an option to map somewhere in the frSetup page mainly to validate that the fields exist during setup